src/packagers/JSPackager.js

```diff
@@ -87,7 +93,12 @@ class JSPackager extends Packager {
       entry.push(this.bundle.entryAsset.id);
     }
 
-    await this.dest.end('},{},' + JSON.stringify(entry) + ')');
+    await this.dest.write('},{},' + JSON.stringify(entry) + ')');
+    await this.dest.write(
+      `\n//# sourceMappingURL=${this.sourcemapPackager.url}`
```

Comment:
Since you won't have access to the source map packager, the URL will be the name of the JS bundle with `.js` replaced with `.map`.

Reply:
Alright, will fix this